client/src/components/ratingsReviews/stars.jsx

```diff
@@ -11,45 +12,10 @@ class Stars extends React.Component {
     calculateStarDiv() {
         //get the rating number passed in 
         let rating = this.props.rating;
-  
-        //get how many full stars there should be by rounding down the nearest num
```

Comment:
I'm assuming that all this is being moved into another file. I think I saw it above